<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Snake</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/images/snake.png</normaloff>:/icon/images/snake.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(233, 243, 255)</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:QColor(0x00,0xff,0x00,0x00)</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>251</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="quitButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QToolButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QToolButton::menu-indicator{image:none;}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QToolButton" name="restartButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QToolButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QToolButton::menu-indicator{image:none;}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="loadButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QToolButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QToolButton::menu-indicator{image:none;}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="startPauseButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QToolButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QToolButton::menu-indicator{image:none;}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="saveButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QToolButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QToolButton::menu-indicator{image:none;}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用时：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="timeLcd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{  
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(138, 240, 255)
}  </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>得分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="scoreLcd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{  
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(138, 240, 255)
}  </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{background-color:white}
QMenu::item:selected{background-color:rgb(162, 131, 255)}</string>
   </property>
   <widget class="QMenu" name="operationMenu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="startAction"/>
    <addaction name="continueAction"/>
    <addaction name="pauseAction"/>
    <addaction name="separator"/>
    <addaction name="loadAction"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="restartAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="optimizationMenu">
    <property name="title">
     <string>优化</string>
    </property>
    <addaction name="setDifficultyAction"/>
    <addaction name="setMusicAction"/>
    <addaction name="showGridAction"/>
    <addaction name="setMapAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpAction"/>
   </widget>
   <addaction name="operationMenu"/>
   <addaction name="optimizationMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="startAction"/>
   <addaction name="continueAction"/>
   <addaction name="pauseAction"/>
   <addaction name="separator"/>
   <addaction name="loadAction"/>
   <addaction name="saveAction"/>
   <addaction name="separator"/>
   <addaction name="restartAction"/>
   <addaction name="quitAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="startAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/027-play button.png</normaloff>:/icon/images/027-play button.png</iconset>
   </property>
   <property name="text">
    <string>开始游戏(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>开始游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/020-download.png</normaloff>:/icon/images/020-download.png</iconset>
   </property>
   <property name="text">
    <string>载入游戏(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>载入游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/022-upload.png</normaloff>:/icon/images/022-upload.png</iconset>
   </property>
   <property name="text">
    <string>保存游戏(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="continueAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/027-play button.png</normaloff>:/icon/images/027-play button.png</iconset>
   </property>
   <property name="text">
    <string>继续游戏(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>继续游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="pauseAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/029-pause.png</normaloff>:/icon/images/029-pause.png</iconset>
   </property>
   <property name="text">
    <string>暂停游戏(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>暂停游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="restartAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/023-refresh.png</normaloff>:/icon/images/023-refresh.png</iconset>
   </property>
   <property name="text">
    <string>重新开始(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>重新开始</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/images/034-error.png</normaloff>:/icon/images/034-error.png</iconset>
   </property>
   <property name="text">
    <string>退出游戏(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>退出游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>帮助文档(&amp;H)</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="setMapAction">
   <property name="text">
    <string>随机地图</string>
   </property>
  </action>
  <action name="showGridAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>栅格：不显示</string>
   </property>
   <property name="toolTip">
    <string>栅格</string>
   </property>
  </action>
  <action name="setMusicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>音乐：关</string>
   </property>
   <property name="toolTip">
    <string>音乐</string>
   </property>
  </action>
  <action name="setDifficultyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>难度：简单</string>
   </property>
   <property name="toolTip">
    <string>难度</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
